subsys/logging/log_backend_net.c

```diff
@@ -199,6 +199,8 @@ static void process(const struct log_backend *const backend,
 {
 	uint32_t flags = LOG_OUTPUT_FLAG_FORMAT_SYSLOG | LOG_OUTPUT_FLAG_TIMESTAMP;
 
+	flags |= IS_ENABLED(CONFIG_LOG_BACKEND_UART_SYST_ENABLE) ? LOG_OUTPUT_FLAG_FORMAT_SYST : 0;
+
```

Comment:
are you sure about it ? This is confuse, because you basically enabling two different formats in this bitmask,
`FORMAT_SYSLOG` and `FORMAT_SYST`.

Reply:
Have removed this from log_backend_net.c because got to know that SYS-T is not supported for network backend.